cmf-cli/Handlers/PackageType/PackageTypeHandler.cs

```diff
@@ -176,6 +176,14 @@ private void GenerateDeploymentFrameworkManifest(DirectoryInfo packageOutputDir)
                         // Each Property in the list will be handled as an xml element with attributes
                         foreach (object property in listProperty)
                         {
+                            // if is Dependency and package type is Test, then skip the dependency on Manifest
+                            // This needs to be changed to get the Type from the dependency list.
+                            // if (property is Dependency &&
+                            //     ((Dependency)property).Type = PackageType.Test)
+                            // {
+                            //     continue;
+                            // }
+
```

Comment:
Remove commented code. Logic for test package is already encapsulated in its respective handler (it does not even generate a manifest)

Reply:
Fixed